app/models/account.rb

```diff
@@ -263,11 +263,11 @@ def bought?(plan)
 
   # TODO: multitenant. enable it?
   # validates_uniqueness_of :s3_prefix
-  validates :org_name, presence: true, length: { maximum: 255 },
-                       format: { with: /\A.*[a-zA-Z0-9]+.*\z/, message: :invalid_format }
+  validates :org_name, presence: true
```

Comment:
Looks like you've lost the `length: { maximum: 255 }` validation? Is it on purpose?

Reply:
Yeah, I added it below, to the list of fields with length validation

Reply:
Ah, nevermind, I see you've moved it to :270